app/views/static/privacy.md

```diff
@@ -13,49 +13,150 @@ Our service allows anyone to check whether they might be eligible to receive qua
+For Apply for QTS in England we rely on:
 
-We use it to determine whether someone might be eligible to receive qualified teacher status (QTS) in England.
+- article 6(1)(e) UK General Data Protection Regulation (UK GDPR), to perform a public task carried out in the public interest as part of our function as a department
+- article 9(2)(g) UK GDPR reasons of substantial public interest (with a basis in law)
+- schedule 1(18) Data Protection Act 2018 (DPA 2018)
 
-## Our legal basis for processing your data
+## The personal data we collect and how we collect it
 
-So that our use of your personal data is lawful, we need to meet one (or more)
-conditions in the data protection legislation. For our service, this is:
+### The data we collect when we check if you’re eligible to apply
 
-- article 6(1)(e) UK General Data Protection Regulation (UK GDPR), to perform a
-  public task carried out in the public interest as part of our function as a
-  department
+We do not collect any personal data about you when you check if you’re eligible to apply
 
-### What we do with your data
+### The data we collect when you make an application for QTS in England
 
-We use the data you give us to determine whether you might be eligible to receive qualified teacher status (QTS) in England. This allows you to apply for teaching jobs or access other DfE services.
+We collect a range of data to assess whether you can be awarded QTS in England:
 
-We might also use your details when analysing data to improve the service.
+- first name
+- last name
+- any previous names (if required)
+- date of birth
+- copies of documents that confirm your - identity such as you passport, driving licence, national identity documents
+- documents that confirm that your name has changed (where appropriate)
+- email address
+- details of your qualifications and copies of documents that confirm these
+- proof of your professional standing as a teacher in the country where you’re recognised.
+- your work history as a teacher
 
-We use GOV.UK PaaS to run the service. [Visit the GOV.UK PaaS website](https://www.cloud.service.gov.uk/privacy-notice/#what-data-we-collect-from-end-users) to find out how they use and look after your data.
+## The data we collect when you confirm the work history of an applicant
+
+If a candidate says that you can confirm their work history, they’ll tell us:
+
+- your name
+- your email address
+- your place of work
+
+## How we use your data
+
+### Processing applications
+
+- Processing applications includes:
+- reviewing your personal data to make a decision about your application for QTS
+- reviewing any further information that is requested from you
+- getting in touch with school leaders you’ve named, to confirm your work history
+- getting in touch if there’s a security issue concerning your data
+- getting in touch with you to ask if you would like to participate in user research
+- analysing application and service usage data
+
+###Safeguarding and fraud
```

Comment:
Should be a space after the #-ses. I don't think it makes a difference to the rendering but easier to read and consistent.